src/RiveQtQuickItem/shader.qrc

```diff
@@ -1,9 +1,3 @@
-<!--
-SPDX-FileCopyrightText: 2023 Jeremias Bosch <jeremias.bosch@basyskom.com>
-SPDX-FileCopyrightText: 2023 basysKom GmbH
-
-SPDX-License-Identifier: LGPL-3.0-or-later
--->

```

Comment:
war das Absicht?

Reply:
yes - this is now handled in a dedicated .license file, since the .qrc file is overwritten by qtcreator when edited.